src/service/service.py

```
@@ -345,70 +313,33 @@ async def message_generator(
-                    # In a more sophisticated implementation, we could add
-                    # some structured ChatMessage type to return the interrupt value.
                     if node == "__interrupt__":
                         interrupt: Interrupt
                         for interrupt in updates:
                             new_messages.append(AIMessage(content=interrupt.value))
                         continue
 
-                    # 비 최종 노드에 대해서는 진행 상황 custom 메시지만 보내고,
-                    # 중간 AI/Human 메시지는 클라이언트에 노출하지 않는다.
                     node_name = str(node).split("/")[-1]
                     progress_line = emit_progress(node_name)
```

Ruff (0.15.0)

[error] Undefined name `emit_progress`

(F821)